go-controller/pkg/ovn/obj_retry.go

```diff
@@ -0,0 +1,118 @@
+package ovn
+
+import (
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+// initRetryObj tracks a failed object to potentially retry later
+// initially it is marked as skipped for retry loop (ignore = true)
+func (r *retryObjs) initRetryObj(obj interface{}, key string) {
+	r.retryMutex.Lock()
+	defer r.retryMutex.Unlock()
+	if entry, ok := r.entries[key]; ok {
+		entry.timeStamp = time.Now()
+		entry.newObj = obj
+	} else {
+		r.entries[key] = &retryObjEntry{newObj: obj, oldObj: nil, config: nil,
+			timeStamp: time.Now(), backoffSec: 1, ignore: true}
+	}
+}
+
+// initRetryWithDelete tracks a failed object to potentially retry later
+// initially it is marked as skipped for retry loop (ignore = true)
```

Comment:
This is the same comment as in `initRetryObj` where we set `newObj`, while here we set `oldObj`. This is more of placeholder for me to think of a better comment for this function :)

Reply:
I fixed the comment to make it more clear